src/opening-period/time-span/TimeSpan.tsx

```diff
@@ -32,6 +31,26 @@ const descriptionPlaceholderTexts: LanguageStrings = {
   en: 'Esim. for women only',
 };
 
+const validateTimeFormat = (timeString: string): boolean => {
+  const [hours, minutes] = timeString.split(':');
+  return (
+    (!hours && !minutes) ||
```

Comment:
Does this mean that the time format is valid if there's no hours and no minutes?

Reply:
I could rename this or something. I agree, the name does not make sense 👋